### ISSUE TITLE
AFC-1807 - Show logotype in the A11y-report

### ISSUE THREAD

**Issue by fredrikarvas-vertel** [vertelab/odoo-af#586]
**Userstory:** 
I want the AF-logotype to be displayed in the A11y-report.
The logo should be visible here: 
See code-line: 
https://github.com/vertelab/odoo-web/blob/48a741ecf8fa7c4d81935cf6ddfd58638aeaaa50/web_a11y_report/static/src/html/tgr.html#L12

I think the logotype needs to be placed in the source-file in the module, and not be inlcuded from Internet because of Odoos security-settings. 

**Module name:** (added by Nils or Fredrik)
https://github.com/vertelab/odoo-web/blob/Dev-12.0/web_a11y_report

**Development Branch name:** 
Dev-12.0-AFC-1807-a11y-report-fix

**Target Branch name:** 
https://github.com/vertelab/odoo-web/blob/Dev-12.0/web_a11y_report

**Test**
Install the logotype. 
Click on Accessibility report - (top right corner)
See the AF-logo

**Description to the manifest:** 
This version makes the logotype visible

**Estimate:** (@developers: Add your times here, not in comments please)
Estimated Hours: 
Estimated Date: 
Actual Hours:

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas I will check this issue.

**Comment by rupareliyahemangi145** [comment]
Hello @FredrikArvas 

I've created PR https://github.com/vertelab/odoo-web/pull/62 for this issue.
Please check it and let me know if anything.

Thanks.

Regards,
Hemangi.

**Comment by fredrikarvas-vertel** [comment]
@erikrudhe Did you see how this change was implemented?
If yes, please close this issu.
